components/api/src/main/java/com/hotels/styx/api/HttpRequest.java

```diff
@@ -183,21 +217,45 @@ public static Builder patch(String uri, StyxObservable<ByteBuf> body) {
         return new Builder(PATCH, uri).body(body);
     }
 
+    /**
+     * Returns the HTTP protocol version.
+     * @return
+     */
     @Override
     public HttpVersion version() {
         return this.version;
     }
 
+    /**
+     * Returns all HTTP headers as {@link HttpHeaders} instance.
+     *
+     * @return {@link HttpHeaders} object.
```

Comment:
In Javadoc convention, the text after a tag does not end with a full stop (`.`). Presumably, this is because the text is not a full sentence.

Reply:
Same comment applies to the rest.